src/detect.ts

```diff
@@ -61,6 +63,20 @@ export function detectSync(options: DetectOptions = {}): DetectResult | null {
   return null
 }
 
+/**
+ * Detects the package manager used in the running process.
+ *
+ * This method will check for `process.env.npm_config_user_agent`.
+ */
+export function getUserAgent(): AgentName | null {
+  const userAgent = process.env.npm_config_user_agent
+  if (!userAgent)
+    return null
```

Comment:
This makes it a bit more readable for less experienced people

```suggestion
  if (!userAgent) {
    return null
  }
```